### ISSUE TITLE
Add "year-over-year" or similar annotation to make trend timespan more obvious

### ISSUE THREAD

**Issue by mthong** [codeforsanjose/economic-indicators-dashboard#87]

**Comment by golgistudio** [comment]
I thought that the trend values are point in time specific.  Do they really represent year over year change?

Also - year-over-year seems to be language for investors.   Will this be readily understood by your primary target audience?
http://www.investopedia.com/terms/y/year-over-year.asp

**Comment by juliemeridian** [comment]
I had a chance to think through this and have a few suggestions: https://www.figma.com/file/YINuNVi5uN3iDdYKlkm2XDgP/CodeForSanJose-Economic-Indicators 

1. Instead of "15-Jun" say "Jun 2015" – the dates get a little lost in the numbers, and the two-digit abbreviation for years can be mistaken for days.
2. Move the rate-of-change arrow and value to the right so these can be side by side when room allows. It looks good to make the arrow a little bigger in this arrangement too.
3. Add a label under the rate of change to clarify when the change was measured from in simple terms ("more than x", "less than x"). I'm not entirely sure how these are measured, so... 
   3a. If it's year-to-year for all of them (e.g. Jun 2014 - Jun 2015, or Q1 2015 - Q1 2016) it's understandable enough to just say the year. 
   3b. If some are _not_ year-to-year (e.g. Q4 2015 - Q1 2016) then use the same level of specificity as the other dates.
4. This would be nice to have: assign colors to changes (the arrow & number) to see at a glance when changes are positive or negative. Green for good, red for bad, and blue for anything that doesn't lend itself to a clear value judgment. I made a few guesses on my design about what I think those might be.

**Comment by golgistudio** [comment]
Julie,

Great comments!  Looks like there are several different issues that you are proposing.  Some of it belongs as part of issue #84

It is helpful to separate issues out, so that they don't get lost.

**Comment by mthong** [comment]
Replies and updates:
@golgistudio - Yes, the trends represent year-over-year changes. For values that are updated every month or quarter, I compare to the same month/quarter of the previous year. For values that are updated annually, I compare to the previous year. However, I agree that "year-over-year" may not be readily understandable to some visitors, so simpler wording is preferable. 

@juliemeridian - great suggestions, thank you!
1. Completely agree. The date formatting has been fixed for the indicators although some of the chart files still need fixing. 

2. I like your suggestions for repositioning the trend arrows and text. I agree with Laurie that it should be listed as a separate Github issue. 

3. I like your suggestion of labeling the period of change using "more than x" and "less than x". However, that required more logic which I didn't have the time to fiddle with in preparation for a Monday demo to city council members, so for now I've simply added "from previous year"

4. In theory I think color coding the arrows would be cool, but in practice I'm concerned about the challenge of placing a value judgment on the changes. Having a neutral color is a good idea but seems to dilute the usefulness of the colors, and can also be seen as a cop-out by partners. For example some colleagues will think that housing prices rising is negative, whereas others think it's a positive sign that our area is in demand.
